js/sort.js

```diff
@@ -0,0 +1,59 @@
+import {getRandomPositiveInteger} from './utils.js';
+const initSubmitSort = (arr, render)=>{
+
+  const filterDiscussed = document.querySelector('#filter-discussed');
+  const filterRandom = document.querySelector('#filter-random');
+  const filterDefault = document.querySelector('#filter-default');
+  const cloneArr = arr;
+
+  const removeElement = ()=>{
+    const picture = document.querySelectorAll('.picture');
+    picture.forEach((element) => {
+      element.remove();
+    });
+  };
+
+  const sortByPopularity = () => {
```

Comment:
Можешь прямо сюда добавить механизм стилизации "активной" кнопки
```suggestion
  const sortByPopularity = () => {
    cloneArr.sort((a, b) => b.comments.length - a.comments.length);
    render(cloneArr);
    // Подсвечивай кнопку сортировки по популярности вверху
  };
```